src/pages/ItemsList/components/ListContent/components/ContentList/ContentList.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import CONTENTS_MOCK from './mockData/contentMock';
+import './contentList.scss';
+
+function ContentList(props) {
+  return (
+    <div className="contentItems">
+      {CONTENTS_MOCK.map(content => {
```

Comment:
```suggestion
      {CONTENTS_MOCK.map(({img, itemName, dragon, type, colors, discountPrice, originalPrice}) => {
```
해당 부분은 위와 같이 구조분해 할당해서 사용할 수 있겠습니다.

Reply:
완료